server/routers/auth.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const path = require('path');
+require('dotenv').config({ path: path.join(__dirname, '../.env') });
+const { Types } = require('mongoose');
+const { User, Cart } = require('../models');
+const { signToken, verifyToken } = require('../jwt');
+
+const auth = express.Router();
+const { ObjectId } = Types;
+
+auth.post('/register', (request, response) => {
+  if (!request.body) {
+    return response.sendStatus(400);
+  }
+
+  const { email, password } = request.body;
+  const user = new User({
```

Comment:
@IgorChaikin Basically you should never trust data, sent from client. You should validate all data on server. For example, I can create user with `email = Vasa`, which is not valid

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/12561302/132222743-f4a0e7ef-81d9-486f-a4f4-f46b1c2375e7.png">